docs/CHANGES.md

```diff
@@ -0,0 +1,59 @@
+# Changes
+
+This document is a place to track all fo the changes that we
+believe must be addressed in other sections of the OCI in
+order to enable the efforts of the WG.
+
+## Distribution Spec
+
+The following is a list of changes may need to be
+addressed in [OCI Distribution Spec](https://github.com/opencontainers/distribution-spec)
+in order to support the goals of this WG:
+
+### Pull
+
+- [add item here]
+
+### Push
+
+- [add item here]
+
```

Comment:
Suggest grouping this into "Storage". I think "Pull and Push" are rudimentary operations already supported by OCI.

Reply:
I disagree. These workflows are already categories in dist spec, and would make this easy to graft any changes required by the WG into these categories.

Reply:
Wouldn't that be something to add to the "upgrade path" document though? As far as user stories are concerned "storage" maps to our "refridgerated truck driver" persona.

Reply:
This doc is mean to be about specific changes that must land in the specs (as needed), vs. user stories or personas. This section may very well end up blank.

Reply:
I guess I don't understand the correspondence between the google doc and these placeholder docs. We haven't decided in the google doc whether we are going with these titles or the ones in the google doc. If we are syncing the two then we ought to do accurately. At this time, there are more community eyes on the google doc than over here. Perhaps this is a topic for discussion in the next meeting (Feb 8 2022).

Reply:
Mostly an attempt here to get more content into this repo so we can progress. Let's chat more tomorrow 👍